post_installation_configuration/cluster-tasks.adoc

```diff
@@ -589,6 +589,9 @@ include::modules/nodes-pods-pod-disruption-configuring.adoc[leveloffset=+2]
 
 [id="post-install-rotate-remove-cloud-creds"]
 == Rotating or removing cloud provider credentials
+
+// If we include removal of resources with ccoctl here, does this text need a rework for covering both "credentials" and "resources"?
```

Comment:
Ccoctl is for installing in STS mode. The only thing worth rotating in this scenario is the private key for signing the service accounts. The creds in STS mode already auto expire/refresh. And rotating the private signing key isn't presently supported.

Reply:
Since this is intended specifically for cleanup after uninstall, I think I'm gonna find a better home for it than here in any case :)